src/deriver/tom/conversational.py

```
@@ -4,9 +4,12 @@
 from anthropic import Anthropic
 from anthropic.types import MessageParam
 from langfuse.decorators import langfuse_context, observe
-from sentry_sdk.ai.monitoring import ai_track
+from anthropic.types import MessageParam, TextBlock
```

Ruff (0.11.9)

Redefinition of unused `MessageParam` from line 5

Remove definition: `MessageParam`

(F811)